packages/@expo/server/src/index.ts

```diff
@@ -1,21 +1,21 @@
 import './install';
```

Comment:
Separate PR (not relevant for this PR): We should aim to remove or move this side-effect away from the entrypoint. It's not relevant for workers and makes it harder to share logic. All responsibilities inside of it don't technically lie with `@expo/server`:
- the sourcemap file support is Node-only and hence should be called in `@expo/cli` or any CLI we add to `@expo/server` for Node
- the `undici` polyfills are likely obsolete, and we can add a higher minimum Node version requirement (I'm pretty sure the embedded undici version will be unproblematic in a version >20)